To: vim-dev@vim.org
Subject: Patch 7.3.033
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.033 (after 7.3.032)
Problem:    Can't build without FEAT_LOCALMAP.
Solution:   Add an #ifdef. (John Marriott)
Files:	    src/getchar.c


*** ../vim-7.3.032/src/getchar.c	2010-10-20 21:22:17.000000000 +0200
--- src/getchar.c	2010-10-22 22:09:40.000000000 +0200
***************
*** 5086,5092 ****
--- 5086,5096 ----
  			if (mp_ptr != NULL)
  			    *mp_ptr = mp;
  			if (local_ptr != NULL)
+ #ifdef FEAT_LOCALMAP
  			    *local_ptr = local;
+ #else
+ 			    *local_ptr = 0;
+ #endif
  			return mp->m_str;
  		    }
  		}
*** ../vim-7.3.032/src/version.c	2010-10-20 21:22:17.000000000 +0200
--- src/version.c	2010-10-22 21:18:30.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     33,
  /**/

-- 
Futility Factor: No experiment is ever a complete failure - it can always
serve as a negative example.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
